blog/_posts/2021-04-15-affils-update.md

```diff
@@ -0,0 +1,137 @@
+
+**Tribal Colleges and Universities (TCUs)**
+
+We have also added affiliation IDs for a partial list of Tribal Colleges and Universities (TCUs), primarily those in the United States that are currently represented in the ADS Literature.  The list of identified TCUs can be accessed [here](https://github.com/adsabs/CanonicalAffiliations/blob/master/tribal_colleges_universities.tsv); note that this list includes only those institutions designated as tribally-controlled, and does not include those public or private non-Tribal institutions having a significant Native American population; some of those schools are present in the list as public universities, however, mainly those that have science and engineering programs.
+
+There are comparatively few publications by authors affiliated with these colleges: Salish Kootenai College (Montana), Northwest Indian College (Pacific NW), Din&eacute; College (Arizona) and Fond du Lac Tribal College (Minnesota) have the most publications. Papers written by TCU authors and coauthors in the ADS primarily focus on topics in Environmental Science and Science Education, which are not currently part of our core literature.  These institutions may become more frequently represented if our scope widens to include more related sciences (e.g. planetary and allied literature).  We also want to ensure that their contributions are noted in existing literature, and that their contributions will be more easily tracked if these institutions increase their participation in the ADS core fields of astronomy and physics.
+
+### Improving Affiliation Metadata: You Can Help!
+
+Affiliation identification is an ongoing process and always will be.  One of our tasks over the past year has been to fix errors in our affiliations metadata, including badly-parsed affiliations; we focused on papers having either the longest author list or the highest citations, but we know there are still some data that needs to be fixed.  Likewise, there are many cases where an author has multiple affiliations and our representation is correct but can't be cleanly split into independent entities -- think of an author who lists "Departments of Chemistry and Physics, University of..." where we assigned IDs to both departments.  We may identify such strings as one or the other, depending on which department we searched for first.  In these cases, our ID is technically correct, but incomplete.
+
+If you find affiliation IDs that aren't correct, please let us know.  Likewise if there are affiliations that you think should be included or better disambiguated as parents and children, please get in touch.  We would especially welcome help with the assignment and disambiguation of French UMRs and Universities, the Russian and Chinese Academies of Science, and any institutions from the Global South especially if they conduct published research in the ADS core fields of Astronomy and Physics.
```

Comment:
I like this specific call for action/help! Add a link to the adshelp@ email somewhere in this paragraph.